### ISSUE TITLE
TypeScript Build Failure: global.d.ts(233,5): error TS2709: Cannot use namespace 'JestConfig' as a type.

### ISSUE THREAD

**Issue by mmichaelis** [playwright-community/jest-playwright#808]
**Describe the bug**

We just tried migrating to jest-playwright 2.0.0, especially because of Jest 28 upgrade, as provided by #801. Unfortunately, build fails with:

```
.../jest-playwright-preset@2.0.0_qruclkpwe5tgscr3gxibacztji/node_modules/jest-playwright-preset/types/global.d.ts(233,5):
  error TS2709: Cannot use namespace 'JestConfig' as a type.
```

**To Reproduce**

Update to Jest-Playwright 2.0.0, use it with TypeScript and don't enable compiler option `skipLibCheck` (see [TypeScript: TSConfig Reference](https://www.typescriptlang.org/tsconfig#skipLibCheck)).

See also at [CodeSandbox](https://codesandbox.io/s/jest-playwright-jestconfig-type-bug-k7dsn9?file=/index.ts):

![CodeSandbox Example](https://user-images.githubusercontent.com/71911/183823808-8f229cc9-9fdd-47a7-8b57-da62544a10f7.png)

**Expected behavior**

Build passes.

**Desktop (please complete the following information):**

 - OS: Windows
 - Playwright version: 2.0.0
 - jest-playwright version: 1.24.2
 - Jest: 28.1.3
 - `@types/jest`: 28.1.6"

**Comment by tbouffard** [comment]
This seems to be fixed in f839747d2e5cb2beeb5de095eddd04229cff6aa0 (commited right after the 2.0.0 release), but there is currently no available release including this fix.

**Comment by tbouffard** [comment]
The fix is included in https://github.com/playwright-community/jest-playwright/releases/tag/v3.0.0.
